### ISSUE TITLE
Is classification logit or probability used as input for temperature scaling?

### ISSUE THREAD

**Issue by WenjianHuang93** [EFS-OpenSource/calibration-framework#32]
I run the classification examplar code for CIFAR dataset, I find the .npz files used store the classification probability instead of the classification logit. Does it mean there is an discrepancy between the original temperature alrogithm [1] and this reproduced algorithm? Thanks for your explanation.

[1] Chuan Guo, Geoff Pleiss, Yu Sun and Kilian Q. Weinberger: "On Calibration of Modern Neural Networks." Proceedings of the 34th International Conference on Machine Learning-Volume 70. JMLR. org, 2017. Get source online <https://arxiv.org/abs/1706.04599>`_

**Comment by fabiankueppers** [comment]
The temperature scaling method consumes probabilities but converts it back to logits internally.
Please refer to #28 for further details.
